scripts/globals/weaponskills/ruinator.lua

```diff
@@ -19,23 +19,25 @@ require("scripts/globals/weaponskills")
 local weaponskill_object = {}
 
 weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
-
     local params = {}
     params.numHits = 4
     params.ftp100 = 1.08 params.ftp200 = 1.08 params.ftp300 = 1.08
-    params.str_wsc = 0.0 + (player:getMerit(xi.merit.RUINATOR) * 0.17) params.dex_wsc = 0.0 params.vit_wsc = 0.0 params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0 params.chr_wsc = 0.0
+    params.str_wsc = 0.0 + (player:getMerit(xi.merit.RUINATOR) * 0.17) params.dex_wsc = 0.0 params.vit_wsc = 0.0
```

Comment:
Could be changed to:
`params.str_wsc = player:getMerit(xi.merit.RUINATOR) * 0.17`